src/components/CartItem.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button, Stack } from "react-bootstrap";
+import { useShoppingCartContext } from '../context/ShoppingCartContext';
+import storeItems from "../data/items.json";
+import { productImageStyles } from "../style/styles";
+
+type CartItemProps = {
+  id: number;
+  quantity: number;
+};
+
+export const CartItem: React.FC<CartItemProps> = ({ id, quantity }) => {
+  const { removeFromCart } = useShoppingCartContext();
+  const item = storeItems.find((i) => i.id === id);
+  if (!item) return null;
+
+  return (
+    <Stack direction="horizontal" gap={2} className="d-flex align-items-center">
+      <img
+        src={item.imgUrl}
+        style={productImageStyles as React.CSSProperties}
```

Comment:
You should add `React.CSSProperties` type on the `productImageStyles` object instead of asserting the type here.